integration-tests/src/user/runtime_user.rs

```diff
@@ -194,6 +194,7 @@ impl RuntimeUser {
         transactions
     }
 
+    // todo get rid of tons of copy pasted code, it's usually outdated comparing to the original
```

Comment:
Add number of specific issue like `TODO(#1234)` so someone else could pick it up.